### PR TITLE
Small OData fixes

### PR DESCRIPTION
**Pull request by biyeun**
##### SUMMARY
- makes sure the over-limit pop up shows when clicking on copy & edit odata feeds
- fixes unique number issue with large / paginated odata feeds

### CODE REVIEW

File: corehq/apps/export/templates/export/partials/odata_feed_limit_reached_modal.html [hunk 1]
```diff
@@ -19,6 +19,11 @@ <h4 class="modal-title">
           Please consider deleting the feeds that you don't need anymore.
         {% endblocktrans %}
       </div>
+      <div class="modal-footer">
+        <button type="button" class="btn btn-default" data-dismiss="modal">
+          Ok
```
Comment (czue) [inline corehq/apps/export/templates/export/partials/odata_feed_limit_reached_modal.html]: Should have a translation tag, no?

File: corehq/apps/api/odata/serializers.py [hunk 2]
```diff
@@ -77,7 +83,7 @@ def serialize_documents_using_config(documents, config, table_id):
         for row_number, document in enumerate(documents):
             rows = table.get_rows(
                 document,
-                row_number,
+                row_number + (self.offset * self.limit),
```
Comment (czue) [inline corehq/apps/api/odata/serializers.py]: nit: might be clearer if this was a property/method. e.g. `self.get_full_row_number()` or something. or even if `(self.offset * self.limit)` was pulled out into `self.start_doc_count` or something...

**Review by czue — COMMENTED** [review]
Thanks, one thing on translations but otherwise G2G. Was using the real IDs was too complicated or did that get vetoed by product?

**Comment by czue** [comment]
Tests failling:
```
======================================================================
ERROR: corehq.apps.api.odata.tests.test_serializers:TestODataCaseSerializer.test_case_id
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/api/odata/tests/test_serializers.py", line 169, in test_case_id
    0
TypeError: serialize_documents_using_config() missing 1 required positional argument: 'table_id'
```

**Comment by biyeun** [comment]
Thanks, @czue! responded

**Review by czue — APPROVED** [review]

**Pull request merged**